node/runtime/pangolin/src/pallets/vesting.rs

```diff
@@ -5,7 +5,7 @@ use sp_runtime::traits::ConvertInto;
 use crate::{weights::pallet_vesting::WeightInfo, *};
 
 frame_support::parameter_types! {
-	pub const MinVestedTransfer: Balance = 100 * MILLI;
+	pub const MinVestedTransfer: Balance = COIN;
```

Comment:
Is this expected? @HackFisher

Reply:
Transfer 0.1 vested COIN is meaningless.

Reply:
If the token's price is 10$, then I think 0.1 is reasonable.

Reply:
![image](https://user-images.githubusercontent.com/11801722/184302993-05f48cad-e0ba-4796-8afa-01863d9ae751.png)

It seems this const is only used here for validation. It won't introduce other issues.

Reply:
LGTM.